java/api-test/integration-tests/pom.xml

```diff
@@ -50,18 +50,18 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.microsoft.semantic-kernel.connectors</groupId>
-            <artifactId>semantickernel-connectors</artifactId>
+            <groupId>com.microsoft.semantic-kernel</groupId>
+            <artifactId>semantickernel-connectors-ai-openai</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.microsoft.semantic-kernel.extensions</groupId>
-            <artifactId>semantickernel-sequentialplanner-extension</artifactId>
+            <groupId>com.microsoft.semantic-kernel</groupId>
+            <artifactId>semantickernel-planning-sequential-planner</artifactId>
```

Comment:
Why wouldn't all planners be in the same artifact (as are the plugins)?

Reply:
I think one module is fine, as you said this will be a very small module.

Reply:
merged these to `semantickernel-planners`